docs/Translation.md

```diff
@@ -0,0 +1,78 @@
+---
+description: BTCPay Translation features.
+---
+
+# Using BTCPay Translation Feature to Localize Your BTCPay Server instance
+
+
+BTCPay Server now includes a powerful translation feature that allows merchant administrators manage their instance in their preferred language. 
+
+This guide will help you create and manage translations, making the BTCPay platform more user-friendly for you.
+
+With this feature, you can replace the default English text across the entire application with your chosen language. 
+
+Whether you want to customize texts on your checkout page, settings configuration, dashboard, or even error and success messages, BTCPay's translation feature makes it easy to tailor your experience to your linguistic needs.
+
+Note: You must be a server admin to access this feature. If you're not an administrator, please introduce this functionality to your admin.
+
+
+## Step-by-Step Guide to Translating Text in BTCPay Server
+
+1. Log in to your BTCPay Server instance.
+2. Navigate to Translation on **Server Settings** >> **Translations**.
+
+![Translation 1](./img/Translations/01_Translation.png)
+
+3. Click **Create** button to generate a new language dictionary.
+
+4. Enter the name you want to use to save the translation dictionary and then click **Create** button to create the dictionary. 
```

Comment:
Enter the name you want to use **to save the translation dictionary**  

Not sure what this means?